docs/keeping_reconciliation_loops_short.md

```diff
@@ -0,0 +1,26 @@
+# Keeping Reconciliation Loops Short
```

Comment:
This is just one point we can document. I want to keep the scope of this PR small and learn as we go.